ui/admin/app/routes/scopes/scope/session-recordings/index.js

```diff
@@ -4,5 +4,160 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
 
-export default class ScopesScopeSessionRecordingsIndexRoute extends Route {}
+export default class ScopesScopeSessionRecordingsIndexRoute extends Route {
+  // =services
+  @service store;
+  @service router;
+  @service can;
+
+  // =attributes
+
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+    times: {
+      refreshModel: true,
+      replace: true,
+    },
+    users: {
+      refreshModel: true,
+      replace: true,
+    },
+    scopes: {
+      refreshModel: true,
+      replace: true,
+    },
+    targets: {
+      refreshModel: true,
+      replace: true,
+    },
+    page: {
+      refreshModel: true,
+    },
+    pageSize: {
+      refreshModel: true,
+    },
+  };
+
+  allSessionRecordings;
+
+  /**
+   * Load all session recordings.
+   * @return {Promise<{ totalItems: number, sessionRecordings: [SessionRecordingModel], sessionRecordingsExist: boolean, storageBucketsExist: boolean }>}
+   */
+  async model({ search, times, users, scopes, targets, page, pageSize }) {
+    const scope = this.modelFor('scopes.scope');
+    const { id: scope_id } = scope;
+    let sessionRecordings;
+    let totalItems = 0;
+    let sessionRecordingsExist = false;
+    let storageBucketsExist = false;
```

Comment:
in the other PR we changed the pattern to something like `dostorageBucketsExist`

Reply:
Right. I just saw that in your PR. I will update now.